studio-server/pom.xml

```diff
@@ -17,6 +17,13 @@
   </properties>
 
   <repositories>
+    <repository>
+      <id>look-here-first</id>
+      <url>https://repo1.maven.org/maven2/</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
```

Comment:
Why is this necessary for this PR? (same for pluginRepositories below)
Is it just for faster builds, or is it required for the GitHub action?
If needed, maybe add a comment here?

Reply:
It's just optimization. I added a comment: [9925a9a](https://github.com/CoreMedia/content-hub-adapter-rss/pull/6/commits/9925a9ac1ab550c925b2e631412b5e60cf7a088f)